src/pages/resultsView/enrichments/GeneBarPlot.tsx

```diff
@@ -177,10 +177,9 @@ export default class GeneBarPlot extends React.Component<IGeneBarPlotProps, {}>
             width = 600;
         }
         return (
-            <div data-test="ClinicalTabPlotDiv" className="borderedChart" style={{ width, overflow: "hidden" }}>
-                <ScrollBar style={{ top: -5 }} getScrollEl={this.getScrollPane} />
+            <div data-test="ClinicalTabPlotDiv" className="borderedChart" style={{ overflowX:'auto', overflowY:'hidden', flexGrow:1  }}>
                 {this.toolbar}
-                <div ref={this.assignScrollPaneRef} style={{ position: "relative", display: "inline-block" }}>
+                <div style={{ position: "absolute" }}>
```

Comment:
this is necessary to get the container to "collapse" around the chart inside of it